iris_grib/__init__.py

```diff
@@ -145,7 +145,6 @@ class GribWrapper(object):
     def __init__(self, grib_message, grib_fh=None):
         """Store the grib message and compute our extra keys."""
         self.grib_message = grib_message
-        self.realised_dtype = np.array([0.]).dtype
```

Comment:
API change

Reply:
Well, not if we never released the previous version, no ? ;-)